tests/pages/component/task-comment-list.js

```diff
@@ -0,0 +1,32 @@
+import {
+  collection,
+  clickable
+} from 'ember-cli-page-object';
+import taskPage from 'code-corps-ember/tests/pages/project/tasks/task';
+import createCommentForm from 'code-corps-ember/tests/pages/components/create-comment-form';
+export default ({
+
+  ClickSave: clickable('.save'),
+
+  comments: collection({
+    scope: '.task-comment-list',
+    itemScope: '.comment-item'
+  }),
+
+  commentBody: {
+    scope: '.comment-body p'
+  },
+
+  createCommentForm,
+
+  taskPage,
```

Comment:
The `taskPage` is the parent here, so you might want to rethink the ordering of your page objects here.